Robot2018/src/org/usfirst/frc/team199/Robot2018/commands/RunScript.java

```diff
@@ -20,15 +22,30 @@ public RunScript(String scriptName) {
     			String cmdName = cmd[0];
     			String cmdArgs = cmd[1];
     			
+    			double[] point = new double[2];
+    			double rotation = 0;
+    			String parentheseless;
+    			String[] pointparts;
+    			if (AutoUtils.isDouble(cmdArgs)) {
+        			rotation = Double.valueOf(cmdArgs);
+        		} else if (AutoUtils.isPoint(cmdArgs)) {
+        			parentheseless = cmdArgs.substring(1, cmdArgs.length() - 1);
+        			pointparts = parentheseless.split(",");
+        			point[0] = Double.parseDouble(pointparts[0]);
+        		}
+    			
     			switch (cmdName) {
 	    			case "moveto":
 	    				addSequential(new AutoMoveTo(cmdArgs.split(" ")));
 	    				break;
 	    			case "turn":
-	    				addSequential(new AutoTurn(Double.parseDouble(cmdArgs)));
+	    				addSequential(new PIDTurn(Double.parseDouble(cmdArgs), Robot.dt, Robot.dt.getGyro()));
+	    				AutoUtils.setRot(rotation);
 	    				break;
 	    			case "move":
-	    				addSequential(new AutoMove(Double.parseDouble(cmdArgs)));
+	    				addSequential(new PIDMove(Double.parseDouble(cmdArgs), Robot.dt, new PIDSourceAverage(null, null)));
+	    				AutoUtils.setX(point[0]);
```

Comment:
instead of creating a new PIDSourceAverage here, make a get method in drivetrain and use that